at_onboarding_cli/functional_tests/pubspec.yaml

```diff
@@ -13,10 +13,7 @@ dependencies:
 
 dependency_overrides:
   at_lookup:
-    git:
-      url: https://github.com/atsign-foundation/at_libraries.git
-      path: at_lookup
-      ref: trunk
+    path: ../../at_lookup
```

Comment:
Don't forget to remove this dependency override after this pull request has been merged to trunk

Reply:
@gkc: Just a thought, Can we leave dependency overrides with path so that for subsequent pushes, at_lookup will be picked from the current branch

Reply:
Why?

Reply:
Just that function tests will run the current branch instead of trunk branch. We can the issue when we raise PR, instead of knowing then after we merge to trunk branch

Reply:
Ah sorry, yes of course that makes sense since it's all in the same repo and we're talking about the functional tests.